src/main/java/com/blamejared/crafttweaker/impl_native/event/entity/player/ExpandPlayerDestroyItemEvent.java

```diff
@@ -0,0 +1,36 @@
+package com.blamejared.crafttweaker.impl_native.event.entity.player;
+
+
+import org.openzen.zencode.java.ZenCodeType;
+
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import com.blamejared.crafttweaker_annotations.annotations.NativeTypeRegistration;
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.event.entity.player.PlayerDestroyItemEvent;
+
+/**
+ * This event is fired when a player destroys a tool or item. This happens
+ * after the tool has been destroyed and can not be used to prevent the
+ * destruction directly.
+ * 
+ * @docParam this event
+ */
+@ZenRegister
+@Document("vanilla/api/event/entity/player/MCPlayerDestroyItemEvent")
+@NativeTypeRegistration(value = PlayerDestroyItemEvent.class, zenCodeName = "crafttweaker.api.event.entity.player.MCPlayerDestroyItemEvent")
+public class ExpandPlayerDestroyItemEvent {
+	
+	/**
+	 * Gets a snapshot of the item from before it broke. Modifying this item
+	 * will have no effect and it should be treated as unmodifiable.
+	 * 
+	 * @return The original item from before it was broken.
+	 */
+    @ZenCodeType.Method
+    @ZenCodeType.Getter("destroyedItem")
+    public static ItemStack getDestroyedItem(PlayerDestroyItemEvent internal) {
+        return internal.getOriginal();
```

Comment:
IItemStacks are a bit of a special case in terms of natives, can you change this to return an IItemStack instead (just wrap `internal.getOriginal()` in a `new MCItemStack(internal.getOriginal()`

This also solves the mutability issue as `MCItemStack` is immutable.